platform/apple/xp/GoldenGateXP.xcodeproj/project.pbxproj

```diff
@@ -2644,7 +2641,6 @@
 		020000000000000000000412 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				BuildIndependentTargetsInParallel = YES;
```

Comment:
Is this change intended?

Reply:
It looks like XCode automatically added that when I saved the project (I didn't change anything explicitly). Do you think that's an issue?

Reply:
Xcode likes to clean things up, probably `BuildIndependentTargetsInParallel = YES` is the default setting now, we should be fine